ghost/core/core/frontend/helpers/ghost_head.js

```diff
@@ -86,7 +86,8 @@ function getSearchHelper(frontendKey) {
     const attrs = {
         key: frontendKey,
         styles: stylesUrl,
-        'sodo-search': adminUrl
+        'sodo-search': adminUrl,
+        locale: settingsCache.get('lang') || 'en'
```

Comment:
This key needs to be `locale` rather than `lang` please.

Reply:
Yeah, I think I can change one word today. Maybe while the tea is steeping. ;)